tests/ui/pages/login.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+"""Page object model for login."""
+
+from pypom import Page
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+
+
+class LoginPage(Page):
+    """Page object model for the login."""
+
+    URL_TEMPLATE = '/{org}/login/'
+
+    @property
+    def title(self):
+        """Return the page title."""
+        self.wait.until(lambda s: self.selenium.title)
+        return self.selenium.title
+
+    @property
+    def alert(self):
+        """Return the alert element."""
+        return self.wait.until(expected.visibility_of_element_located((
+            By.CSS_SELECTOR,
+            ".alert-danger",
+        )))
```

Comment:
Returning elements means the test author needs to be familiar with the Selenium client, which the page object model helps to abstract. Instead, you could return the text of the alert. If there are other attributes of the alert that you might want to access, then consider a `Region` with methods/properties.